app/views/comments/_form.html.erb

```diff
@@ -6,6 +6,7 @@
   <%= f.text_area :body %> <br>
   <%= f.hidden_field :commentable_id, value: comment.commentable.id %>
   <%= f.hidden_field :commentable_type, value: comment.commentable.class %>
+  <%= f.hidden_field :ancestry unless comment.ancestry.blank? %>

```

Comment:
Is this a good idea? It's trivial to mess with hidden fields content with Chrome web inspector, for example. Won't your user be able to hack/break the app by messing with this?